spec/tasks/gen/resource_spec.cr

```diff
@@ -37,6 +37,26 @@ describe Gen::Action do
     end
   end
 
+  it "allows for all supported column types" do
+    with_cleanup do
+      io = generate Gen::Resource::Browser,
+        "Alphabet",
+        "a:Bool",
+        "b:Int16",
+        "c:Int32",
+        "d:Int64",
+        "e:String",
+        "f:UUID",
+        "g:Time",
+        "h:Float64",
+        "i:JSON::Any",
+        "j:Array(String)"
+      io.to_s.should_not contain("Must provide valid columns")
+      should_generate_migration named: "create_alphabets.cr", with: "add i : JSON::Any"
+      should_generate_migration named: "create_alphabets.cr", with: "add j : Array(String)"
```

Comment:
I wonder if there's a better way that I can check all of these columns and only run the expectation once? 🤔